src/components/list-podcast/script.js

```diff
@@ -50,10 +50,7 @@ export default {
       default: '100%',
     },
     to: {
-      type: String,
-    },
-    detailPath: {
-      type: String,
+      type: Object,
```

Comment:
1. **to** property could have both types, [String, Object]
2. Are we using  **to** property somewhere?

Reply:
1.  **To** prop will always receive an object
2. To will always be used in the the navigation of the button More